src/components/KTable/KTable.vue

```diff
@@ -93,7 +99,7 @@
               :class="{ 'is-scrolled': isScrolled }"
             >
               <th
-                v-for="(column, index) in tableHeaders"
+                v-for="(column, index) in visibleHeaders"
                 :key="`k-table-${tableId}-headers-${index}`"
```

Comment:
can we use the column instead of the index as the key?

Reply:
I will change this but it's old code. Realistically this could wait until the reskin to fix?

Reply:
You can skip for now